drivers/dma/dma_mcux_lpc.c

```diff
@@ -487,9 +562,21 @@ static int dma_mcux_lpc_get_status(const struct device *dev, uint32_t channel,
 
 static int dma_mcux_lpc_init(const struct device *dev)
 {
+	const struct dma_mcux_lpc_config *config = dev->config;
 	struct dma_mcux_lpc_dma_data *data = dev->data;
 	int total_dma_channels;
 
+	if (!data->data_cb || !data->otrig_array) {
```

Comment:
This block needs to be removed, we do not need to check if `data_cb` and `otrig_array` are NULL as they are not allocated

Reply:
Removed thank you.